release/air_tests/air_benchmarks/workloads/benchmark_util.py

```diff
@@ -115,3 +117,14 @@ def run_fn_on_actors(
     for actor in actors:
         futures.append(actor.run_fn.remote(fn, *args, **kwargs))
     return ray.get(futures)
+
+
+def time_it(f):
+    @wraps(f)
+    def wrapper(*args, **kwargs):
+        start = time.monotonic()
+        f(*args, **kwargs)
+        time_taken = time.monotonic() - start
+        return time_taken
+
+    return wrapper
```

Comment:
maybe we can just use the default python timeit

https://docs.python.org/3/library/timeit.html#python-interface